lib/matplotlib/axis.py

```diff
@@ -1438,9 +1438,17 @@ def grid(self, b=None, which='major', **kwargs):
 
         """
         if len(kwargs):
+            if not b and b is not None:  # something not-None falsey
+                warnings.warn('First parameter to grid() is falsey, but line '
```

Comment:
Do you just mean 'False'?  "falsey" is not something I'm familiar with, though maybe you mean False-like?